js/app/modules/arrangement/sideBySide.js

```diff
@@ -3,40 +3,61 @@
 /* exported SideBySide */
 
 const Endless = imports.gi.Endless;
+const Gdk = imports.gi.Gdk;
 const Gtk = imports.gi.Gtk;
 
 const Arrangement = imports.app.interfaces.arrangement;
 const Knowledge = imports.app.knowledge;
 const Module = imports.app.interfaces.module;
+const Utils = imports.app.utils;
 
 const MENU_HEIGHT = 50;
-const _HorizontalSpacing = {
-    TINY: 15,
-    SMALL: 20,
-    LARGE: 40,
-    XLARGE: 50,
-};
+const SPACING_MIN = 15;
+const SPACING_MAX = 50;
 
 const _SideBySideLayout = new Knowledge.Class({
     Name: 'Arrangement.SideBySideLayout',
-    Extends: Endless.CustomContainer,
+    Extends: Gtk.Box,
 
     _init: function (props={}) {
         this.all_visible = true;
+        props.orientation = Gtk.Orientation.HORIZONTAL;
 
         this.parent(props);
-    },
 
-    // Removing a visible widget should recalculate the positions of all widgets
-    vfunc_remove: function (widget) {
-        let needs_resize = widget.get_child_visible();
-        this.parent(widget);
-        if (needs_resize)
-            this.queue_resize();
-    },
+        /* Button to trigger dropdown */
+        this._popover_button = new Gtk.Button({ label: " ⁝ " });
```

Comment:
The new design uses the ⁝ (tricolon )
https://endlessm.invisionapp.com/d/main#/console/5300658/128774538/preview